README.md

```diff
@@ -1 +1,151 @@
 # ampersand-multifield-view
+
+## purpose
+A view module for intelligently grouping multiple field views. Works well with [ampersand-form-view][ampersand-form-view].
```

Comment:
if we can elaborate on this a bit we can better communicate that it is the primary purpose of the module; organizing multi-level form data objects versus the traditional single-level grab bag(`FormView`).